indico/modules/events/registration/models/items.py

```diff
@@ -437,6 +437,10 @@ def view_data(self):
     def _get_default_log_data(self):
         return {'Section ID': self.id}
 
+    @make_interceptable
+    def is_visible(self, is_management):
+        return not self.is_deleted and (is_management or not self.is_manager_only)
```

Comment:
This is a potential change in behavior (not implying that the old behavior is more or less correct). Previously, the `is_deleted` check was only applied outside management, ie in management a deleted section would still be shown in the registration info.

Reply:
> This is a potential change in behavior (not implying that the old behavior is more or less correct). Previously, the `is_deleted` check was only applied outside management, ie in management a deleted section would still be shown in the registration info.

Sorry, original logic restored.